packages/www/components/Dashboard/StreamDetails/StreamHealthTab.tsx

```diff
@@ -43,13 +59,16 @@ const StreamHealthTab = ({ stream, streamHealth, invalidateStream }) => {
         return;
       }
       const newInfo = rinfo as StreamInfo;
+
       setDataChart((prev) => {
         const lastItem = prev[prev.length - 1];
         return [
           ...prev,
           {
             name: lastItem ? lastItem.name + ingestInterval / 1000 : 0,
```

Comment:
Oh I see, so you are actually not creating these specs on this PR right. Don't mind refactoring it since it's not a regression, but it'd be great to improve these interfaces a bit anyway if you're open to it.